src/modules/layout/components/ModalHeader.js

```diff
@@ -56,19 +41,16 @@ class Header extends React.PureComponent<PropsType, StateType> {
   }
 
   render () {
-    const headerTitle = this.getDescriptor().headerTitle || ''
-
     return (
       <BoxShadow theme={this.props.theme}>
         <Horizontal>
           <HorizontalLeft>
             <HeaderBackButton onPress={this.goBack} />
```

Comment:
the null in goBack is acually required by design: https://reactnavigation.org/docs/en/navigation-prop.html#goback-close-the-active-screen-and-move-back